server/proxy/notify.go

```diff
@@ -0,0 +1,193 @@
+package proxy
+
+import (
+	"encoding/json"
+	"path"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/upstream"
+	"github.com/mattermost/mattermost-plugin-apps/utils"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/pkg/errors"
+)
+
+func (p *Proxy) Notify(cc *apps.Context, subj apps.Subject) error {
+	subs, err := p.store.Subscription.Get(subj, cc.TeamID, cc.ChannelID)
+	if err != nil {
+		return err
+	}
+
+	return p.notify(cc, subs)
+}
+
+func (p *Proxy) NotifyRemoteWebhook(app *apps.App, data []byte, webhookPath string) error {
```

Comment:
This function is unchanged, and is only part of this diff because it moved files.